pixel-saver@deadalnix.me/schemas/org.gnome.shell.extensions.pixel-saver.gschema.xml

```diff
@@ -0,0 +1,9 @@
+<schemalist>
+	<schema id="org.gnome.shell.extensions.pixel-saver" path="/org/gnome/shell/extensions/pixel-saver/">
+		<key name="show-buttons" type="b">
```

Comment:
That's a good idea. My approach would have been to use two switches

```
Show window buttons in top bar                 [On/Off]
    Place buttons next to application menu     [On/Off]
```
The second only available when the first switch is active.

But having to deal with only one option would certainly simplify the code. I will try that. Thanks.